examples/material-ui-nextjs-pages-router-ts/src/Link.tsx

```diff
@@ -39,10 +25,9 @@ export const NextLinkComposed = React.forwardRef<HTMLAnchorElement, NextLinkComp
         shallow={shallow}
         passHref
```

Comment:
It looks like we can remove `passHref` now per https://nextjs.org/docs/app/api-reference/components/link and the source: https://github.com/vercel/next.js/blob/8e0acd90341d59f53447189f265c5c8f9b1e3c28/packages/next/src/client/link.tsx. 

I'm fixing this in https://github.com/mui/material-ui/pull/44871.